modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Edges/ObjectRelationshipEdge.tsx

```diff
@@ -3,18 +3,12 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-import ClayDropDown from '@clayui/drop-down';
-import ClayIcon from '@clayui/icon';
-import React, {useEffect, useRef, useState} from 'react';
-import {EdgeProps, EdgeText} from 'react-flow-renderer';
+import React, {useEffect, useState} from 'react';
+import {EdgeProps} from 'react-flow-renderer';
 
-import {useObjectFolderContext} from '../ModelBuilderContext/objectFolderContext';
-import {TYPES} from '../ModelBuilderContext/typesEnum';
 import {ObjectRelationshipEdgeData} from '../types';
-import ManyMarker from './ManyMarker';
-import OneMarker from './OneMarker';
-
-import './Edge.scss';
+import {ManyObjectRelationshipEdge} from './ManyObjectRelationshipEdge';
+import {SimpleObjectRelationshipEdge} from './SimpleObjectRelationshipEdge';
```

Comment:
about the names `ManyObjectRelationshipEdge` and `SimpleObjectRelationshipEdge` i think they don't complement each other well, for example better opposites would be:

`ManyObjectRelationshipEdge` - `OneObjectRelationshipEdge`
`ManyObjectRelationshipEdge` - `SingleObjectRelationshipEdge`
`SimpleObjectRelationshipEdge` - `ComposedObjectRelationshipEdge`

it's not essential to change this now since the PR is very crucial and needs to be on the main branch asap, but i think we can improve these names later

Reply:
Its a good suggestion @rebsilva, thanks! We can do this in a technical debit :smile: